ingestion/src/metadata/profiler/profiler/core.py

```diff
@@ -406,7 +407,7 @@ def compute_metrics(self) -> Profiler:
 
         return self
 
-    def process(self, generate_sample_data: Optional[bool]) -> ProfilerResponse:
+    def process(self, source_config) -> ProfilerResponse:
```

Comment:
@NiharDoshi99 here only the required params can be passed instead of the entire `source_config`